nerd-icons.el

```diff
@@ -471,6 +471,10 @@
     ;; Homebrew
     ("^Brewfile$"              nerd-icons-faicon "nf-fa-beer"                :face nerd-icons-lsilver)
 
+    ;; Arch
+    ("^PKGBUILD$"              nerd-icons-flicon "nf-linux-archlinux"        :face nerd-icons-cyan)
```

Comment:
Hi, I just want to know why not use `nf-dev-archilinux`?

Reply:
No reason, they look identical to me. If there's a reason to prefer `nf-dev-archlinux`, I'm happy to switch.